src/index.js

```diff
@@ -12,16 +12,21 @@ export default function (elem, centerIfNeeded, options, finalElement) {
       }
   }
 
+  const offsetTop = config.offsetTop || 0;
+  const offsetLeft = config.offsetLeft || 0;
+  const offsetBottom = config.offsetBottom || 0;
+  const offsetRight = config.offsetRight || 0;
```

Comment:
This is a breaking change, since using `config` like this assumes that it has been passed in in line 3 above! Right now, the current version is broken and not usable...

Reply:
You're right, it'll be fixed right away: https://github.com/stipsan/scroll-into-view-if-needed/pull/129